gdal/apps/gdal_rasterize_bin.cpp

```diff
@@ -48,7 +48,7 @@ static void Usage(const char* pszErrorMsg = NULL)
         "       [-co \"NAME=VALUE\"]* [-a_nodata value] [-init value]*\n"
         "       [-te xmin ymin xmax ymax] [-tr xres yres] [-tap] [-ts width height]\n"
         "       [-ot {Byte/Int16/UInt16/UInt32/Int32/Float32/Float64/\n"
-        "             CInt16/CInt32/CFloat32/CFloat64}] [-q]\n"
+        "             CInt16/CInt32/CFloat32/CFloat64}] [-optim {[AUTO]/VECTOR/RASTER}] [-q]\n"
```

Comment:
This should also be documented in apps/gdal_utilities.dox and with explanations of what VECTOR and RASTER imply